doc/manipulating_images/data_preparation.rst

```diff
@@ -214,122 +261,91 @@ Common data preparation steps: resampling, smoothing, filtering
-downsample image quality on purpose to increase processing speed
-and lower memory consumption of an analysis pipeline.
-In fact, certain image viewers (e.g., FSLView) also require images to be
-resampled to display overlays.
+Smoothing
+.........
 
-On an advanced note, automatic computation of offset and bounding box
-can be performed by specifying a 3x3 matrix instead of the 4x4 affine.
-In this case, nilearn computes automatically the translation part
-of the transformation matrix (i.e., affine).
+:class:`NiftiMasker` can apply Gaussian spatial smoothing to the
+neuroimaging data, useful to fight noise or for inter-individual
+differences in neuroanatomy. It is achieved by specifying the full-width
+half maximum (FWHM; in millimeter scale) with the `smoothing_fwhm`
+parameter. Anisotropic filtering is also possible by passing 3 scalars
+``(x, y, z)``, the FWHM along the x, y, and z direction.
 
-.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_affine_transformation_002.png
-    :target: ../auto_examples/04_manipulating_images/plot_affine_transformation.html
-    :scale: 33%
-.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_affine_transformation_004.png
-    :target: ../auto_examples/04_manipulating_images/plot_affine_transformation.html
-    :scale: 33%
-.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_affine_transformation_003.png
-    :target: ../auto_examples/04_manipulating_images/plot_affine_transformation.html
-    :scale: 33%
+The underlying function handles properly non-cubic voxels by scaling the
+given widths appropriately.
 
+.. seealso::
 
-.. topic:: **Special case: resampling to a given voxel size**
+   :func:`nilearn.image.smooth_img`
 
-   Specifying a 3x3 matrix that is diagonal as a target_affine fixes the
-   voxel size. For instance to resample to 3x3x3 mm voxels::
+.. _temporal_filtering:
 
-    >>> import numpy as np
-    >>> target_affine = np.diag((3, 3, 3))
+Temporal Filtering and confound removal
+........................................
 
-.. topic:: **Resampling an image to a given reference image**
+:class:`NiftiMasker` can also improve aspects of temporal data
+properties, before conversion to voxel signals.
 
-    Instead of using an affine or shape, it's also possible to resample an
-    image using a reference image with :func:`nilearn.image.resample_to_img`.
+- **Standardization**. Parameter ``standardize``: Signals can be
+  standardized (scaled to unit variance). 
 
-.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_resample_to_template_001.png
-    :target: ../auto_examples/04_manipulating_images/plot_resample_to_template.html
-    :scale: 33%
+- **Frequency filtering**. Low-pass and high-pass filters can be used to
+  remove artifacts. Parameters: ``high_pass`` and ``low_pass``, specified
+  in Hz (note that you must specific the sampling rate in seconds with
```

Comment:
perhaps give an example with concrete numbers, people might worry about how to specify that exactly